source/_patterns/30-organisms/collection/collection.scss

```diff
@@ -25,12 +25,79 @@ $background-overlap-wide: 50px;
     background-color: transparent;
     box-shadow: none;
   }
+
+  &:first-child {
+    &.collection--branded {
+      .collection__branding-wrapper {
+        padding-top: 16px;
+
+        @include tablet-up {
+          padding-top: 32px;
+        }
+
+        @include desktop-wide {
+          padding-top: 40px;
+        }
+      }
+    }
+
+    &.collection--borderless {
+      .collection__branding-wrapper {
+        padding-top: 0;
+      }
+    }
+  }
+
+  &:last-child {
+    margin-bottom: 16px;
+
+    &.collection--branded {
+      margin-bottom: 0;
+
+      .collection__branding-wrapper {
+        padding-bottom: 16px;
+      }
+    }
+
+    @include tablet-up {
+      margin-bottom: 32px;
+
+      &.collection--branded {
+        margin-bottom: 0;
+
+        .collection__branding-wrapper {
+          padding-bottom: 32px;
+        }
+      }
+    }
+
+    @include desktop-wide {
+      margin-bottom: 40px;
+
+      &.collection--branded {
+        margin-bottom: 0;
```

Comment:
diese zeile ist wohl obsolet, weil in zeile 55 und tablet-up bereits definiert

Reply:
-> Spezifizität mit MQ 🤔

Reply:
![image](https://user-images.githubusercontent.com/909895/52791978-85256d80-306a-11e9-8f8f-36ab57debbda.png)